front/pages/api/stripe/webhook.ts

```diff
@@ -368,6 +368,7 @@ async function handler(
               status: "ended",
               endDate: new Date(),
             });
+            await revokeUsersForDowngrade(activeSubscription.workspace.id);
```

Comment:
Could we pass the `WorkspaceType` instead of the id directly? We should have it now that we have the auth object 🙏🏻

Reply:
You think it's required even for private functions, local to the file? (I put it in a function because I think it's more readable but I could have inlined it )

Asking because the auth object is in another switch block not this one (otherwise I would fully agree), I have to reconstruct it here if I want the workspace type and I tend to think it's not worth the added 3-4 lines block

Reply:
What I've understand is we never send the "id" and always use the "sId" unless we don't have a choice, and I think  
Of course feel free to merge as is if you disagree with this.

But here as we already have access to `activeSubscription.workspace` it seems that we'd only have to add:

```
const auth = await Authenticator.internalBuilderForWorkspace(
  activeSubscription.workspace.sId
);
const owner = auth.workspace();
await revokeUsersForDowngrade(owner);
await sendOpsEmail(owner.sId);
```

instead of
```
await revokeUsersForDowngrade(activeSubscription.workspace.id);
await sendOpsEmail(activeSubscription.workspace.sId);
```


Of course feel free to merge as is if you prefer as your code work 🙇🏻‍♀️

Reply:
Thanks for your opinion :pray: We agree it would add a 4 line block to the code

IMHO,the function here is only a more readable equivalent of inlining, in which case the question would not have been asked: workspace.id is used directly in many other places in the file webhook.ts, so the rule doesn't strictly applies. In that case, I think adding 4 lines is not worth it.